src/helpers/exampleRunner.ts

```diff
@@ -0,0 +1,16 @@
+import dotenv from 'dotenv';
```

Comment:
Same here - should be in `examples/helpers`

Reply:
done